examples/02-Multi-GPU-Tensorflow-with-Horovod.ipynb

```diff
@@ -0,0 +1,371 @@
+{
```

Comment:
<p>Why is Horovod in brackets? [Horovod]? Do you want to add a link for it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/dataloader/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>yes, apologies, that was the intention, fixed it now</p>